### DISCUSSION TITLE
`restartPolicy: Always` for initContainers

### DISCUSSION THREAD

**Discussion by marklonquist** [argoproj/argo-workflows#13477] [Q&A]
Is this not possible? We use this "sidecar" pattern for our normal workloads

**Comment by agilgur5** [comment]
It seems like no, but I couldn't figure out why as the code doesn't touch container-level `restartPolicy`.
See this prior discussion on it: https://github.com/argoproj/argo-workflows/discussions/12826#discussioncomment-9542958
